authbridge/authlib/plugins/lineage/plugin_test.go

```
@@ -0,0 +1,912 @@
+	pctx.Extensions.MCP = &pipeline.MCPExtension{
+		Method: "message/send",
+		Params: map[string]any{"message": map[string]any{"role": "user"}},
+		Result: map[string]any{"artifacts": []any{map[string]any{"artifactId": "a-1"}}},
+	}
+
+	run(t, p, pctx, allow(200))
+	req, resp := roleSplit(t, exp.GetSpans())
+
+	checkAttr(t, req, "lineage.protocol", "a2a")
+	if v, ok := findAttr(req, "input.value"); ok {
+		t.Errorf("input.value = %q on an a2a hop with no a2a parts — leaked from the co-populated MCP parse", v.Emit())
```

golangci-lint (2.12.2)

[error] SA1019: v.Emit is deprecated: Use [Value.String] instead. 

(staticcheck)